RecoTracker/MkFitCMS/BuildFile.xml

```diff
@@ -0,0 +1,5 @@
+<use name="RecoTracker/MkFitCore"/>
+<flags CXXFLAGS="-fopenmp-simd"/>
```

Comment:
for my education, does this imply restrictions on the CPU family and architecture? Does it affect e.g. the PowerPC and ARM builds of CMSSW?

Reply:
No, it should not ... it just enables `#pragma omp simd` to request compier assisted loop vectorization.